Actors/CalculatorActor/build.ps1

```diff
@@ -6,11 +6,11 @@ popd
 pushd src\CalculatorActorApplication\CalculatorActor\
 dotnet restore -s https://api.nuget.org/v3/index.json 
 dotnet build 
-dotnet publish -o ..\..\..\CalculatorActorApplication\CalculatorActorPkg\Code
+dotnet publish -c Release --self-contained --runtime win-x64 -o ..\..\..\CalculatorActorApplication\CalculatorActorPkg\Code

```

Comment:
The more obvious and portable way is to update the project file with these two properties instead of the --self-contained and --runtime switches.

<PublishSelfContained>true</<PublishSelfContained>
<RuntimeIdentifier>win-x64</RuntimeIdentifier>

Reply:
The self contained switch could be added, thanks.

For the RuntimeIdentifier, the same csproj is used for Linux & Windows builds.

Reply:
Ok makes sense otherwise it'll just require more overhead to have the runtime within the project file.